contrib/python-pytest-env/template.py

```diff
@@ -0,0 +1,23 @@
+pkgname = "python-pytest-env"
+pkgver = "1.1.3"
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "python-build",
+    "python-installer",
+    "python-hatchling",
+    "python-hatch_vcs",
+    "python-wheel",
```

Comment:
you can leave wheel out when hatchling is used

Reply:
Okay will take out